c_src/build_deps.sh

```diff
@@ -7,15 +7,15 @@ TARGET="$PRIV_PATH"/libsecp256k1_nif.so
 
 case "$1" in
   clean)
-    rm -rf secp256k1
+    rm -rf c_src/secp256k1
     ;;
 
   *)
     # skip download and compile when the compiled so is there
     test -f $TARGET && exit 0
-    (test -d c_src/secp256k1 || git clone https://github.com/bitcoin/secp256k1)
+    (test -d c_src/secp256k1 || cd c_src && git clone https://github.com/bitcoin/secp256k1)
 
-    (cd c_src/secp256k1 && git reset --hard 5a91bd768faaa974e00301e662fd8f2aa75a122a &&  ./autogen.sh && ./configure --enable-module-recovery && make)
+    (cd c_src/secp256k1 && git reset --hard 5a91bd768faaa974e00301e662fd8f2aa75a122a &&  ./autogen.sh && ./configure --enable-openssl-tests=no --enable-module-recovery && make)
```

Comment:
should we try now to update this to the head of master (`b19c000063be11018b4d1a6b0a85871ab9d0bdcf`)?

Reply:
I tried it @leishman but it broke quite a lot of thing. So I guess it will take some effort that's why I didn't update this in this sprint

Reply:
ah ok got it.